sunpy/util/tests/test_util.py

```diff
@@ -48,11 +54,30 @@ def test_expand_list():
     lst = [1, 2, 3, [4, 5, 6], 7, (8, 9), ((10, 11), ((12, 13),))]
     assert util.expand_list(lst) == [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13]
 
-
-def test_expand_list_generator():
-    lst = ['a', 'b', [], (['c', 'd']), tuple(), ['e']]
-    assert list(util.expand_list_generator(lst)) == ['a', 'b', 'c', 'd', 'e']
-
+@pytest.mark.filterwarnings("ignore:Invalid 'BLANK' keyword in header")
+def test_expand_list_generator(aia171_test_map):
```

Comment:
This test could be cleaned up quite a bit using `parametrize`.

Reply:
Have a look now.

Reply:
Beautiful. Thanks!